draft-ietf-keytrans-protocol.md

```diff
@@ -288,74 +288,96 @@ correctly.
 
 Each leaf node in a prefix tree represents a specific mapping from search key to
 value, while each parent node represents some prefix which all search keys in
-the subtree headed by that node have in common. The subtree headed by a parent's
-left child contains all search keys that share its prefix followed by an
-additional 0 bit, while the subtree headed by a parent's right child contains
-all search keys that share its prefix followed by an additional 1 bit.
-
-The root node, in particular, represents the empty string as a prefix. The
-root's left child contains all search keys that begin with a 0 bit, while the right
-child contains all search keys that begin with a 1 bit.
+the subtree headed by that node have in common. The prefix that a parent node
+represents is based on the its **midpoint** and the **first unspecified
+byte** of the search key.
```

Comment:
The sentence doesn't "type check" for me. A prefix tree's inner node does not represent a single search key, right? It represents a set of search keys. It doesn't work for me to say "what a parent node is based on *the* search key."